src/System.ServiceModel.Shim/System.ServiceModel.Shim.csproj

```diff
@@ -14,30 +14,15 @@
     <!-- The partial facade assembly's public key token must exactly match the contract to be filled. -->
     <StrongNameKeyId>ECMA</StrongNameKeyId>
 
-    <!--allow desktop references-->
-    <AssetTargetFallback>net472</AssetTargetFallback>
-    <NoWarn>$(NoWarn);NU1701</NoWarn>
     <MicrosoftTargetingPackNETFrameworkv472Package>Microsoft.TargetingPack.NETFramework.v4.7.2</MicrosoftTargetingPackNETFrameworkv472Package>
     <IsPackable>false</IsPackable>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="$(MicrosoftTargetingPackNETFrameworkv472Package)" Version="$(MicrosoftTargetingPackNETFrameworkv472PackageVersion)" PrivateAssets="All" />
-    <PackageReference Include="Microsoft.DotNet.GenFacades" Version="$(MicrosoftDotNetGenFacadesPackageVersion)" PrivateAssets="All" />
+    <PackageDownload Include="$(MicrosoftTargetingPackNETFrameworkv472Package)" Version="[$(MicrosoftTargetingPackNETFrameworkv472PackageVersion)]" />
```

Comment:
I know this was a pre-existing version choice, but why 4.7.2? The oldest version of NetFx that supports netstandard2.0 is 4.6.1. That's what we target in our nuget packages, we have a net461 folder.

Reply:
I bet this was the newest at the time. The purpose of this project is to get the largest set of types that “might” exist on .NETCore for building a .NETFramework compat shim.  From that perspective you can update to .NET 4.8. It would only matter if you added types to System.ServiceModel in 4.8 and those types are also present in .NETCore.